pandas/tests/io/test_sql.py

```diff
@@ -1430,6 +1430,28 @@ def test_sqlalchemy_integer_overload_mapping(self, integer):
         ):
             sql.SQLTable("test_type", db, frame=df)
 
+    def test_sqlalchemy_oracle_number_type_mapping(self):
+        from numpy import (
```

Comment:
Instead of doing this just reference these objects from the `np.*` import that is already done in the module